turbo.json

```diff
@@ -0,0 +1,42 @@
+{
+  "$schema": "https://turborepo.org/schema.json",
+  "pipeline": {
+    "//#typecheck:watch": {
+      "cache": false
+    },
+    "//#build:watch": {
+      "cache": false
+    },
+    "//#test:watch": {
+      "dependsOn": ["typecheck", "format:check"],
+      "cache": false
+    },
+    "test:codemods": {
+      "outputs": []
+    },
+    "clean": {
+      "cache": false
+    },
+    "//#test:ci": {
+      "dependsOn": ["format:check", "typecheck"],
+      "outputs": ["coverage/**"]
+    },
+    "test:eslint": {
+      "outputs": []
+    },
+    "//#typecheck": {
+      "outputs": []
+    },
+    "//#format:check": {
+      "outputs": []
+    },
+    "//#build": {
```

Comment:
can you explain or point to the documentation about what this `//#` syntax means? It looks pretty foreign to me 😅

Reply:
Yes, here you go!

https://turborepo.org/docs/core-concepts/running-tasks#running-tasks-from-the-root